thoth/storages/graph/models.py

```diff
@@ -72,7 +72,7 @@ class PythonPackageVersion(Base, BaseExtension):
     )
     is_missing = Column(Boolean, nullable=False, default=False)
     provides_source_distro = Column(Boolean, nullable=False, default=True)
-
+    import_package = Column(Text, nullable=False)
```

Comment:
nullable should be True, initially the database won't have these entries and we cannot assign a default in this case, wdyt?

Reply:
> nullable should be True, initially the database won't have these entries and we cannot assign a default in this case, wdyt?

I agree, corrected in this commit d1a6c6a850b5e8f3e2577648f8c348225f492f8c.
Thanks for pointing!